alert/store.go

```
@@ -791,3 +791,44 @@ func (s *Store) State(ctx context.Context, alertIDs []int) ([]State, error) {
 
 	return list, nil
 }
+
+func (s *Store) Feedback(ctx context.Context, alertID int) (*Feedback, error) {
+	err := permission.LimitCheckAny(ctx, permission.System, permission.User)
+	if err != nil {
+		return nil, err
+	}
+
+	row, err := gadb.New(s.db).AlertFeedback(ctx, int64(alertID))
+	if errors.Is(err, sql.ErrNoRows) {
```

Missing `err != nil` check after this (would be a silent failure)